docs/GettingStarted.md

```diff
@@ -18,23 +18,12 @@ To run ORT the following tools are required:
 * Java (version >= 8)
 * Git
 
-For some of the supported package managers and SCMs additional tools need to be installed:
-
-* [Bundler](http://bundler.io)
-* [CVS](http://savannah.nongnu.org/projects/cvs)
-* [dep](https://github.com/golang/dep) 0.4.1
-* [Glide](https://glide.sh)
-* [Go](https://golang.org) (tested with 1.10)
-* [Godep](https://github.com/tools/godep)
-* [Maven](http://maven.apache.org)
-* [Mercurial](https://www.mercurial-scm.org)
-* [NPM](https://www.npmjs.com) 5.5.* - 6.4.* and [Node.js](https://nodejs.org) 8.* (required for this tutorial because
-  we scan `mime-types` which is an NPM project)
-* [PHP Composer](https://getcomposer.org) >= 1.5
-* [SBT](https://www.scala-sbt.org/)
-* [Stack](https://docs.haskellstack.org/en/stable/README)
-* [Subversion](https://subversion.apache.org)
-* [Yarn](https://yarnpkg.com) 1.3.* - 1.9.*
+For some of the supported package managers and Version Control Systems additional tools need to be installed. In the
```

Comment:
Maybe mention or link the full list of tools in `README.md`.

Reply:
Done.